Frameworks-integrations/MIGraphX/Quantizations.rst

```diff
@@ -0,0 +1,145 @@
+Running Quantized ResNet50 via MIGraphX
+========================================
+
+This example walks through the dynamo PTQ workflow for running a quantized model using torch_migraphx.
+
+Prerequisite 
++++++++++++++++
+
+You must follow the `installation instructions <https://github.com/ROCmSoftwarePlatform/torch_migraphx>`_ before using this example. 
+
+Steps for running a quantized model using torch_migraphx
++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++
+
+1. Use torch.export and quantize_pt2e APIs to perform quantization
+------------------------------------------------------------------
+
+Note: The export API call is considered a prototype feature at the time
+this tutorial is written. Some call signatures may be modified in the
+future.
+
+.. code:: ipython3
+
+    import torch
+    from torchvision import models
+    from torch._export import capture_pre_autograd_graph
+    from torch.ao.quantization.quantize_pt2e import prepare_pt2e, convert_pt2e
+    
+    import torch_migraphx
+    from torch_migraphx.dynamo.quantization import MGXQuantizer
+
+
+.. code:: ipython3
+
+    model_fp32 = models.resnet50(weights=models.ResNet50_Weights.IMAGENET1K_V1).eval()
+    input_fp32 = torch.randn(2, 3, 28, 28)
+    
+    torch_fp32_out = model_fp32(input_fp32)
+
+The ``capture_pre_autograd_graph`` call will be changed to a
+torch.export.export call once it supports the pre autograd capture
+functionallity. Currently there is a known issue when using only kwargs
+as inputs. See: https://github.com/pytorch/pytorch/issues/113744
+
+.. code:: ipython3
+
+    model_export = capture_pre_autograd_graph(model_fp32, (input_fp32, ))
+
+Use the pt2e API to prepare, calibrate, and convert the model
+~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+
+Torch-MIGraphX provides a custom Quantizer for performing quantization that is compatible with MIGraphX. 
+
+Note: Additional configs will also work as long as the configs ensure symmetric quantization using the signed int8
+datatype. Currently, only symmetric quantization is supported in MIGraphX.
+
+.. code:: ipython3
+
+    quantizer = MGXQuantizer()
+    m = prepare_pt2e(model_export, quantizer)
+
+.. code:: ipython3
+
+    # psudo calibrate
+    with torch.no_grad():
+        for _ in range(10):
+            m(torch.randn(2, 3, 28, 28))
+
+.. code:: ipython3
+
+    q_m = convert_pt2e(m)
+    torch_qout = q_m(input_fp32)
+
+2. Lower Quantized model to MIGraphX
```

Comment:
The meaning of this title does not make sense to me. import lower quantized model, ingest lower quantized model, build lower quantized model?

Reply:
"Lower" here is used as a verb. Its a commonly used term in the context of optimizing deeplearning models on specific hardware. It means this step is "lowering" the model from the framework level to the graph compiler level